nistats/first_level_model.py

```diff
@@ -183,9 +183,9 @@ class FirstLevelModel(BaseEstimator, TransformerMixin, CacheMixin):
         This parameter specifies the desired drift model for the design
         matrices. It can be 'polynomial', 'cosine' or None.
 
-    period_cut : float, optional
-        This parameter specifies the cut period of the high-pass filter in
-        seconds for the design matrices. Used only if drift_model is 'cosine'.
+    high_pass : float, optional
+        This parameter specifies the cut frequency of the high-pass filter in
+        H for the design matrices. Used only if drift_model is 'cosine'.
```

Comment:
in Hz

Reply:
Same here. This small fix was addressed [in my latest commit](https://github.com/nistats/nistats/pull/351/commits/b80487a30e8bc523592345d953571c68ed2d76c7).